### ISSUE TITLE
请教预测任务的学习率衰减策略

### ISSUE THREAD

**Issue by Sunyingbin** [PaddlePaddle/Paddle#36308]
我在做回归任务，对能源信息进行预测，learing_rate取一个定值觉得不太好，想用一个学习率衰减方式，请问应该用哪个合适？求帮忙！
https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/optimizer/Overview_cn.html#about-lr

**Comment by gongel** [comment]
你好，常用的有`LinearWarmup`和`NoamDecay`。

**Comment by Sunyingbin** [comment]
> LinearWarmup

learning_rate=0.001
scheduler = paddle.optimizer.lr.LinearWarmup(
        learning_rate=learning_rate, warmup_steps=5, start_lr=0, end_lr=0.000001, verbose=True)

训练中，控制台一直打出Epoch 116615: LinearWarmup set learning rate to 0.001.这样
learing rate并没有递减，一直是初始值，这是怎么回事请问

**Comment by gongel** [comment]
你好，scheduler需要放在优化器中哈。

**Comment by Sunyingbin** [comment]
> 你好，scheduler需要放在优化器中哈。

放了，好像是linearWarmup这个好像是把学习率递增的？

**Comment by Sunyingbin** [comment]
scheduler = paddle.optimizer.lr.NoamDecay(d_model=0.01, warmup_steps=100, verbose=True)
试了NoamDecay，d_model是什么意思

**Comment by gongel** [comment]
你好，可以看下官方文档哈
https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/optimizer/lr/LinearWarmup_cn.html#linearwarmup
https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/optimizer/lr/NoamDecay_cn.html#noamdecay
